auto_tuning/proxy/src/proxy_bindings.cpp

```diff
@@ -0,0 +1,49 @@
+#include "proxy_common.hpp"
+#include <pybind11/pybind11.h>
+namespace py = pybind11;
+
+ProxyOutput runProxy(ProxyConfig config);
```

Comment:
Why do you not add it to `proxy_common.hpp` or another header?

Reply:
done